study-builder/studies/testboston/study-workflows.conf

```diff
@@ -64,31 +70,23 @@
           "activityCode": ${id.act.covid_survey},
           "expression": """
             user.studies["testboston"].forms["COVID_SURVEY"].hasInstance()
-            && user.studies["testboston"].forms["COVID_SURVEY"].isStatus("CREATED", "IN_PROGRESS")
+            && !user.studies["testboston"].forms["COVID_SURVEY"].isStatus("COMPLETE")
           """
         },
         {
           "type": "ACTIVITY",
           "activityCode": ${id.act.address},
           "expression": """
             user.studies["testboston"].forms["ADDRESS"].hasInstance()
-            && user.studies["testboston"].forms["ADDRESS"].isStatus("CREATED", "IN_PROGRESS")
+            && !user.studies["testboston"].forms["ADDRESS"].isStatus("COMPLETE")
           """
         },
         {
           "type": "ACTIVITY",
           "activityCode": ${id.act.consent},
           "expression": """
             user.studies["testboston"].forms["CONSENT"].hasInstance()
-            && user.studies["testboston"].forms["CONSENT"].isStatus("CREATED", "IN_PROGRESS")
-          """
-        },
-        {
-          "type": "ACTIVITY",
-          "activityCode": ${id.act.consent},
-          "expression": """
-            !user.studies["testboston"].forms["CONSENT"].hasInstance()
-            && user.studies["testboston"].hasInvitation("RECRUITMENT")
```

Comment:
We will no longer implicitly use the Workflow service to create the `consent` when called with `RETURN_USER`. It will be done by events instead, which should be less prone to accidental creations.